locales/hu_HU/pcsx2_Iconized.po

```diff
@@ -5,71 +5,70 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 0.9.8\n"
+"Project-Id-Version: PCSX2 1.6.0\n"
 "Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
 "POT-Creation-Date: 2020-02-10 17:29+0000\n"
-"PO-Revision-Date: 2011-04-17 17:56+0100\n"
-"Last-Translator: Delirious <delirious@freemail.hu>\n"
-"Language-Team: Delirious <delirious@freemail.hu>\n"
-"Language: \n"
+"PO-Revision-Date: 2020-03-18 20:58+0100\n"
+"Last-Translator: dio\n"
+"Language-Team: \n"
+"Language: hu\n"
```

Comment:
should be hu_HU
```suggestion
"Language: hu_HU\n"
```

Reply:
Fixed. No idea why Poedit keeps two separate Hungarian locales, Hungarian is only spoken at Hungary in official capacity.